tests/test_csv.py

```diff
@@ -133,6 +133,74 @@ def test_file_errors(self, contents, error_rows, message):
         if message:
             assert status["error_messages"][0] == message
 
+    def test_column_overrides(self):
```

Comment:
mwah

Reply:
this is for this whole file, not just this line

Reply:
Good mwah or bad mwah?